bagitobjecttransfer/recordtransfer/templates/includes/submission_group_form.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n %}
+<script>
+    // Pass IDs of form fields elements to JavaScript
+    const ID_SUBMISSION_GROUP_DESCRIPTION = "{{ ID_SUBMISSION_GROUP_DESCRIPTION }}";
+    const ID_SUBMISSION_GROUP_NAME = "{{ ID_SUBMISSION_GROUP_NAME }}";
+</script>
```

Comment:
Same suggestion as above:

```suggestion
{% block javascript %}
    {{ block.super }}
    <script>
        // Pass IDs of form fields elements to JavaScript
        const ID_SUBMISSION_GROUP_DESCRIPTION = "{{ ID_SUBMISSION_GROUP_DESCRIPTION }}";
        const ID_SUBMISSION_GROUP_NAME = "{{ ID_SUBMISSION_GROUP_NAME }}";
    </script>
{% endblock %}
```

Reply:
Resolved by:
https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/226/commits/bf1d9d5a012c5a2c091b4604b858c9350c06a4b3